index.html

```diff
@@ -717,16 +728,32 @@ <h2>Overview</h2>
                     alt="Exploration mechanisms high-level class diagram" />
                 <figcaption>
                     The high-level class diagram of the exploration mechanisms,
-                    depicting how Things expose <a>TDs</a>.
+                    depicting how Thing Description Servers and Thing Description Directories provide <a>TDs</a>.
+                    A Self-describing Thing is a special case of a Thing Description Server that is also a Thing.
+                    A Thing Description Directory hosts a Thing Description Server for each 
+                    Thing Description it contains.
```

Comment:
It doesn't really host a server for each TD. Literally doing so will be very inefficient.
```suggestion
                    A Thing Description Directory acts as a Thing Description Server for each 
                    Thing Description it contains.
```

Reply:
need a better word than "server", is really an endpoint